Gemfile

```diff
@@ -3,14 +3,20 @@ source 'https://rubygems.org'
 # The gem's dependencies are specified in the gemspec
 gemspec
 
-group :local_development do
-  gem 'pry'
-  gem 'yard', '~> 0.8.7'
+group :test do
+  gem 'aruba',         '~> 0.10.0'
+  gem 'ataru',         '~> 0.2.0'
+  gem 'cucumber',      '~> 2.0'
+  gem 'factory_girl',  '~> 4.0'
+  gem 'rake',          '~> 10.0'
+  gem 'rspec',         '~> 3.0'
+  gem 'rubocop',       '~> 0.34.0'
+end
 
+group :development do
```

Comment:
I think I preferred you original development/debugging proposal. With test/development, combined with how the groups are named in Rails, I get confused why rake would be in test.

Oh, maybe the gems in test don't actually need to be in a group?